libminifi/include/Connection.h

```diff
@@ -180,7 +184,8 @@ class Connection
 	std::atomic<uint64_t> _maxQueueDataSize;
 	//! Flow File Expiration Duration in= MilliSeconds
 	std::atomic<uint64_t> _expiredDuration;
-
+	//! UUID string
+	std::string _uuidStr;
```

Comment:
all the other private members in the Connection.h are like _XXX based. Just want to keep it consistent.